pyproject.toml

```diff
@@ -30,21 +25,71 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
 ]
 
+[tool.poetry]
+name = "robyn"
+version = "0.36.2"
+description = "A web server that is fast!"
+authors = ["Sanskar Jethi <sansyrox@gmail.com>"]
+repository = "https://github.com/sansyrox/robyn"
+documentation = "https://robyn.tech/"
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: BSD License",
+  "Operating System :: OS Independent",
+  "Topic :: Internet :: WWW/HTTP",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+]
 
-[project.urls]
-"Source Code" = "https://github.com/sansyrox/robyn"
-Issues = "https://github.com/sansyrox/robyn/issues"
-Documentation = "https://robyn.tech/"
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/sansyrox/robyn/issues"
 Changelog = "https://github.com/sansyrox/robyn/blob/main/CHANGELOG.md"
 
-[project.optional-dependencies]
-"templating" = ["jinja2 == 3.0.1"]
+[tool.poetry.dependencies]
+python = "^3.7"
+maturin = "0.14.12"
+watchdog = "2.2.1"
+multiprocess = "0.70.14"
+nestd = "0.3.1"
+uvloop = { version = "0.17.0", markers = "sys_platform != 'win32' and (sys_platform != 'cygwin' and platform_python_implementation != 'PyPy')" }
+jinja2 = { version = "3.0.1", optional = true }
+
+[tool.poetry.extras]
+templating = ["jinja2"]
+
+[tool.poetry.group.dev]
+optional = true
+
+[tool.poetry.group.dev.dependencies]
+ruff = "0.0.240"
+black = "23.1"
+isort = "5.11.5"
+pre-commit = "2.21.0"
+commitizen = "2.40"
+
+[tool.poetry.group.test]
+optional = true
+
+[tool.poetry.group.test.dependencies]
+requests = "2.28.2"
+pytest = "7.2.1"
+websocket-client = "1.5.0"
+pytest-codspeed = "1.2.2"
+
+[tool.poetry.scripts]
+devserver = { callable = "integration_tests.base_routes:main" }
```

Comment:
one final change
can we name it `test_server` or something similar?

We have a dev server for live reloading